diagnostic.md

```diff
@@ -40,31 +47,38 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
 ```sh
-  # < Your Response Here >
+A serializer is thought of as a filter for database information. The serializer
+ determines which columns will show when tables are viewed. Columns that are not
+ shown do still exist however, they just wont be displayed.
 ```
 
 ```rb
 class ProfileSerializer < ActiveModel::Serializer
+  attributes :given_name, :surname, :email
 end
 ```
```

Comment:
There would also be a `has_many :movies` line but this covers the relevant serialized data.